src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -47,6 +47,8 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!-- Runtime feature defaults to trim unnecessary code -->
     <InvariantGlobalization Condition="'$(InvariantGlobalization)' == ''">false</InvariantGlobalization>
+    <InvariantTimezone Condition="'$(BlazorEnableTimeZoneSupport)' == 'false'">true</InvariantTimezone>
```

Comment:
How does `BlazorEnableTimeZoneSupport` get set?
Is `'$(InvariantTimeZone)' == '' and` not be needed here, would `InvariantTimezone` ever be set before this or does `BlazorEnableTimeZoneSupport` override?

Reply:
It's public property, see [Blazor docs](https://learn.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-7.0#disable-unused-features)

`BlazorEnableTimeZoneSupport` overrides any earlier `InvariantTimezone`